module/apps/actor-importer-regexp.js

```diff
@@ -1,269 +1,491 @@
-/* global CONFIG, game */
+/* global game */
 import { CoC7Utilities } from '../utilities.js'
 
```

Comment:
This is probably the most cryptic and complex file of the importer and I think it would be nice to have a general comment at the beginning explaining how it works.